### PR TITLE
Fix page navigation error when delete template.

### PR DESCRIPTION
**Pull request by utchoang**
Open for https://github.com/apache/cloudstack-primate/issues/598
@rhtyd cc @svenvogel I've completed this task. Please review it. Thanks.

### CODE REVIEW

**Comment by rhtyd** [comment]
Lgtm can you review and test @davidjumani

**Comment by rhtyd** [comment]
@blueorangutan package

**Review by davidjumani — APPROVED** [review]
Tested. LGTM

File: src/views/image/TemplateZones.vue [hunk 1]
```diff
@@ -266,7 +266,10 @@ export default {
           jobId,
           successMethod: result => {
             if (singleZone) {
-              this.$router.go(-1)
+              const isResourcePage = !!(this.$route.params && this.$route.params.id)
```
Comment (davidjumani) [inline src/views/image/TemplateZones.vue:269]: @utchoang `!!` Double not ?

Comment (utchoang) [reply src/views/image/TemplateZones.vue]: @davidjumani I've removed it.

**Comment by davidjumani** [comment]
@rhtyd Good to go! Thanks @utchoang

**Pull request merged**